### PR TITLE
JSON type support

### PR DESCRIPTION
**Pull request by cbaron**
Stacked on #12

### CODE REVIEW

File: packages/appsync-emulator-serverless/__test__/example/schema.graphql [hunk 1]
```diff
@@ -1,3 +1,5 @@
+scalar JSON
```
Comment (lightsofapollo) [inline packages/appsync-emulator-serverless/__test__/example/schema.graphql]: Wrong type name: https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html#awsjson

This brings up the issue that we really will need to start building an integration suite for the emulator that validates our test schemas against real appsync to ensure we're not missing details like this.

Comment (cbaron) [reply packages/appsync-emulator-serverless/__test__/example/schema.graphql]: :( 

Good catch.

Hmmm, integration suite.  Solid idea.  How do you see it working?  Deploying the test example using serverless and ensuring the invocation succeeds?

Comment (lightsofapollo) [reply packages/appsync-emulator-serverless/__test__/example/schema.graphql]: Thinking we write a test suite that works against both the emulator and appsync (via api key auth perhaps?)

File: packages/appsync-emulator-serverless/dynamodbSource.js [hunk 2]
```diff
@@ -198,6 +195,17 @@ const scan = async (
     Segment: segment,
     TotalSegments: totalSegments,
   };
+  if (filter) {
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/dynamodbSource.js]: From what I recall dynamo didn't like empty `ExpressionAttributeNames` or `ExpressionAttributeValues`

File: packages/appsync-emulator-serverless/lambdaSource.js [hunk 3]
```diff
@@ -16,8 +21,19 @@ const lambdaSource = async (
...
+      [`DYNAMODB_TABLE_${alias}`]: tableName,
+    }),
+    {},
+  );
 
   const child = fork(Runner, [], {
+    env: {
+      ...dynamodbTableAliases,
+      DYNAMODB_ENDPOINT: dynamodbEndpoint,
+    },
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/lambdaSource.js]: lambda can talk to local dynamo

File: packages/appsync-emulator-serverless/dynamodbSource.js (hunk 2 above)
Comment (lightsofapollo) [reply packages/appsync-emulator-serverless/dynamodbSource.js]: correct!

File: packages/appsync-emulator-serverless/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@conduitvc/appsync-emulator-serverless",
-  "version": "0.6.2",
+  "version": "0.6.8",
```
Comment (lightsofapollo) [inline packages/appsync-emulator-serverless/package.json]: New features mean a version bump to 0.7.0

File: packages/appsync-emulator-serverless/schema.js [hunk 5]
```diff
@@ -307,7 +313,9 @@ const generateResolvers = (cwd, config, configs) => {
         },
       };
     },
-    {},
+    {
```
Comment (lightsofapollo) [inline packages/appsync-emulator-serverless/schema.js]: should use your scalars export from schemaWrapper

**Pull request merged**